### DISCUSSION TITLE
PCBWay as an approved vendor

### DISCUSSION THREAD

**Discussion by Miner16** [hackclub/OnBoard#1360] [Ideas]
Could PCBWay be added to the approved vendors list? I am more familiar with their processes and UI.

**Comment by ghost** [comment]
Yeah I agree with you @blazecoding2009 .

**Comment by blazecoding2009** [comment]
i believe it is, let me check

On Thu, Dec 19, 2024 at 4:46 PM cryptopapi713 ***@***.***>
wrote:

> Yeah I agree with you @blazecoding2009
> <https://github.com/blazecoding2009> .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/OnBoard/discussions/1360#discussioncomment-11622321>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANP2AL2VSWWKEMUM62QLAZL2GM5CHAVCNFSM6AAAAABT245OGWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRSGIZTEMI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Thanks and regards,
Arnnav Kudale

https://blazecoding. <https://blazecoding.me/>xyz
+1 (437) 255 3382

**Reply by Miner16** [reply]
Did you check?

**Reply by Miner16** [reply]
@blazecoding2009

**Reply by Miner16** [reply]
@LimesKey

**Reply by LimesKey** [reply]
No they cannot be

**Comment by LimesKey** [comment]
https://github.com/hackclub/OnBoard/pull/1106#issuecomment-2356000916
